src/backend/distributed/commands/trigger.c

```diff
@@ -714,7 +714,9 @@ ErrorOutForTriggerIfNotSupported(Oid relationId)
 	}
 	else if (IsCitusTableType(relationId, REFERENCE_TABLE))
 	{
-		ereport(ERROR, (errmsg("triggers are not supported on reference tables")));
+		ereport(ERROR, (errmsg("triggers are not supported on reference tables "
+							   "when \"citus.enable_unsafe_triggers\" is set to "
+							   "\"false\"")));
```

Comment:
Let's not put that GUC in error messages. This feature is very experimental and people might not read the manual if they get a GUC from an error message.

We have sometimes seen people enabling this due to it being in the error message and then use triggers that are not correct for pushdown.